src/main/java/mate/academy/bookstore/repository/role/RoleRepository.java

```diff
@@ -1,9 +1,11 @@
 package mate.academy.bookstore.repository.role;
 
+import java.util.Optional;
 import mate.academy.bookstore.model.Role;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
```

Comment:
```suggestion
```

Reply:
I have a role change, I save the roles when registering and that's it, since you didn't tell me to do that before.